src/components/services/Medusa.vue

```diff
@@ -0,0 +1,123 @@
+<template>
+  <div>
+    <div class="card" :class="item.class">
+      <a :href="item.url" :target="item.target" rel="noreferrer">
+        <div class="card-content">
+          <div class="media">
+            <div v-if="item.logo" class="media-left">
+              <figure class="image is-48x48">
+                <img :src="item.logo" :alt="`${item.name} logo`" />
+              </figure>
+            </div>
+            <div v-if="item.icon" class="media-left">
+              <figure class="image is-48x48">
+                <i style="font-size: 35px" :class="['fa-fw', item.icon]"></i>
+              </figure>
+            </div>
+            <div class="media-content">
+              <p class="title is-4">{{ item.name }}</p>
+              <p class="subtitle is-6">{{ item.subtitle }}</p>
+            </div>
+            <div class="notifs">
+              <strong
+                v-if="config !== null && config.system.news.unread > 0"
+                class="notif news"
+                title="News"
+              >{{ config.system.news.unread }}</strong>
+              <strong
+                v-if="config !== null && config.main.logs.numWarnings > 0"
+                class="notif warnings"
+                title="Warning"
+              >{{ config.main.logs.numWarnings }}</strong>
+              <strong
+                v-if="config !== null && config.main.logs.numErrors > 0"
+                class="notif errors"
+                title="Error"
+              >{{ config.main.logs.numErrors }}</strong>
+              <strong
+                v-if="serverError"
+                class="notif errors"
+                title="Connection error to Medusa API, check url and apikey in config.yml"
+              >?</strong>
+            </div>
+          </div>
+          <div class="tag" :class="item.tagstyle" v-if="item.tag">
+            <strong class="tag-text">#{{ item.tag }}</strong>
+          </div>
+        </div>
+      </a>
+    </div>
+  </div>
+</template>
+
+<script>
+export default {
+  name: "Medusa",
+  props: {
+    item: Object,
+  },
+  data: () => {
+    return {
+      config: null,
+      serverError: false,
+    };
+  },
+  created: function () {
+    this.fetchConfig();
+  },
+  methods: {
+    fetchConfig: async function () {
+      this.config = await fetch(
+        `${this.item.url}/api/v2/config`,{
+            credentials: 'include',
+            headers: { 'X-Api-Key': `${this.item.apikey}` }
+        }
+      )
+      .then((response) => {
```

Comment:
I wouldn't use `.then` with async / await like that. You can either drop the async await and assign the config in the `then` function, or drop the `then` / `catch` for a `try` / `catch`. 
`await` makes possible to stop the execution and wait for the result, we usually use it instead of the promise syntax.